libs/shared/utils/src/gtag/reportConversion.ts

```diff
@@ -0,0 +1,13 @@
+export const gtag_report_conversion = (url?: Location | undefined) => {
+  const callback = function () {
+    if (typeof url != 'undefined') {
+      window.location = url;
+    }
+  };
+
+  window.gtag('event', 'conversion', {
+    send_to: `${process.env['NEXT_PUBLIC_GOOGLE_ANALYTICS']}/${process.env['NEXT_PUBLIC_CONVERSION_LABEL']}`,
```

Comment:
I'm not sure it makes any difference but is there any reason you're doing `process.env.['NEXT_PUBLIC_GOOGLE_ANALYTICS']` vs `process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS`?

Reply:
@kherma

Reply:
This is probably where the `undefined`s are coming from. I'm going to remove the quotes and see if that fixes it.

Reply:
Check that: I'm going to add these env variables to Vercel, and see if that fixes it.

Reply:
This implementation choice doesn't affect this PR's behavior, so I'm going to resolve it here. Further discussion can take place outside the repo.